### ISSUE TITLE
IE11 meta tag

### ISSUE THREAD

**Issue by jezbo7** [CDrummond/lms-material#389]
This skin wont work in any IE engine under IE11, could you add this meta tag to the opening page?

`<meta http-equiv="X-UA-Compatible" content="IE=11">`

I'm rendering it in a .NET WebBrowser control which needs this if it depends on IE11 or Edge.

**Comment by CDrummond** [comment]
I've added the tag to master for 2.1.3 (if released) or 2.2.0

**Comment by JeremyBradshaw7** [comment]
Many thanks. I see this is in master but my LMS still showing 2.1.2 
![image](https://user-images.githubusercontent.com/36155348/102876322-f1c6db80-443c-11eb-8cb6-71a34ad68266.png)
Is there a separate release process for rolling this out to the LMS plugin repository?

**Comment by CDrummond** [comment]
That's because I have not made a 2.1.3 or 2.2.0 release yet - I do not make a release after every change. Sorry, but you either need to run a git dev version, or wait for me to make a release.

**Comment by JeremyBradshaw7** [comment]
> That's because I have not made a 2.1.3 or 2.2.0 release yet - I do not make a release after every change. Sorry, but you either need to run a git dev version, or wait for me to make a release.

That's fine thanks, just asking.

**Comment by MetaTunes** [comment]
Thanks for 2.1.3
It works in Edge (as did 2.1.2) but not IE
2 errors listed:
SCRIPT1002: Syntax error
material.min.js (62,40)
SCRIPT5009: 'Vue' is undefined
material-deferred.min.js (232,108)

**Comment by CDrummond** [comment]
All I did was add the tag you requested. Sorry, but I do not use Windows so have no time to investigate issues with IE. If you have an fixes then I will gladly apply, but I have no time for this myself.

**Comment by jezbo7** [comment]
MetaTunes is a different user, not me. He is using my my .NET product though which renders the LMS interface in a WebBrowser. I'll investigate why it's still throwing the javascript errors.

**Comment by jezbo7** [comment]
I'm now thinking if the skin relies on ES2015 constructs like classes, we may need that IE=11 meta tag changed to IE=edge in order for it to load in a .NET WebBrowser control. For optimal IE/Edge compatibility I'd recommend:

`<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />`

**Comment by CDrummond** [comment]
Did you not try these tags before asking for the original change?

**Comment by MetaTunes** [comment]
I've tried <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />. It doesn't work.
@jezbo7 - I suggest I test on my system before suggesting changes

**Comment by jezbo7** [comment]
No sorry didn't try it first, I thought it would help. I think you can remove the meta tag again for your next release as not even IE=edge seems to help either.

**Comment by MetaTunes** [comment]
Just for the record, @jezbo7 fixed Muso to use a modern browser, so the original reason for this request is no longer an issue
